src/audio/pipeline/pipeline-stream.c

```diff
@@ -296,13 +296,15 @@ static int add_pipeline_cps_consumption(struct comp_dev *current,
 		cd = &md->cfg.base_cfg;
 	}
 
-	int kcps = cd->cpc * 1000 / ppl_data->p->period;
-	if (kcps == 0) {
-		tr_warn(pipe, "0 KCPS for module: %#x, core: %d", current->ipc_config.id, ppl_data->p->core);
-	} else {
-		core_kcps_adjust(ppl_data->p->core, kcps);
-		tr_info(pipe, "Registering KCPS consumption: %d, core: %d", kcps, ppl_data->p->core);
+	if (cd->cpc == 0) {
+		cd->cpc = PLATFORM_IPC4_MAX_CPC;
```

Comment:
Is this compiles on no ACE platforms?

It might be better to
```c
/* Use maximum clock budget, assume 1ms chunk size */
cd->cpc = CLK_MAX_CPU_HZ / 1000;
```